server/graphql/v2/interface/Account.js

```diff
@@ -185,10 +185,11 @@ const accountFieldsDefinition = () => ({
         query.where.role = { [Op.in]: roles };
       }
 
-      let conditionOnMemberCollective;
-      if (args.type) {
-        const types = args.type.split(',');
-        conditionOnMemberCollective = { type: { [Op.in]: types } };
+      const conditionOnMemberCollective = {};
+      if (args.accountType && args.accountType.length > 0) {
+        conditionOnMemberCollective.type = {
+          [Op.in]: args.accountType.map(value => AccountTypeToModelMapping[value]),
+        };
```

Comment:
That's right. Useful for for Individual -> User.

Reply:
Keeping it consistent with the existing behaviour for V2 account querying.